### ISSUE TITLE
Warnings from `script/module_library_setup.R`

### ISSUE THREAD

**Issue by eloch216** [biocro/skelBML#6]
When running `script/module_library_setup.R` using `source`, there are sometimes a few warnings that pop up. For example, here is the output after running it on the `update-from-skeleton` branch of the `cgbml` repository, where I have supplied the package name `cgbml`: 

```
moving ../DESCRIPTION to ..\DESCRIPTION-361855ae4d64.bak
wrote ../DESCRIPTION
No changes to ../src/module_library/example_module.h.
No changes to ../src/init.cpp.
No changes to ../LICENSE.md.
moving ../src/module_library/module_library.cpp to ../src/module_library\module_library.cpp-36182b2a39cd.bak
wrote ../src/module_library/module_library.cpp
No changes to ../src/module_library/module_library.h.
No changes to ../NAMESPACE.
moving ../NEWS.md to ..\NEWS.md-3618419d7fdb.bak
wrote ../NEWS.md
moving ../README.md to ..\README.md-36183e2a4e47.bak
wrote ../README.md
No changes to ../src/R_module_library.cpp.
No changes to ../tests/testthat/test.Modules.R.
No changes to ../tests/testthat.R.
No changes to ../tests/module_test_cases/cgbml_example_module.csv.
Warning messages:
1: In readLines(destination) == text :
  longer object length is not a multiple of shorter object length
2: In readLines(destination) == text :
  longer object length is not a multiple of shorter object length
3: In readLines(destination) == text :
  longer object length is not a multiple of shorter object length
4: In readLines(destination) == text :
  longer object length is not a multiple of shorter object length
```

**Comment by gsrohde** [comment]
The warning comes when `==` is used to compare vectors of unequal length where the longer vector is not an integral multiple of the shorter (so that the shorter one can't be recycled an integral number of times).  This likely arose in the cgbml case because the existing files differed significantly from the template files (in particular, having a different number of lines).

I replaced `==` with `identical`, which probably should have been used in any case (see PR #9).

Partly because I forgot this was an issue for `biocro/skellBML` rather than `biocro/cgbml`, I went ahead and changed it on the `update-from-skeleton branch` branch of the latter.
